### PR TITLE
[AIRFLOW-5237] Less verbose output for CI builds

### PR DESCRIPTION
**Pull request by potiuk**
Make sure you have checked _all_ steps below.

### Jira

- [x] My PR addresses the following [Airflow Jira](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"
  - https://issues.apache.org/jira/browse/AIRFLOW-5237

### Description

- [x] Here are some details about my PR, including screenshots of any UI changes:

### Tests

- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits

- [x] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [x] In case of new functionality, my PR adds documentation that describes how to use it.
  - All the public functions and the classes in the PR contain docstrings that explain what it does
  - If you implement backwards incompatible changes, please leave a note in the [Updating.md](https://github.com/apache/airflow/blob/master/UPDATING.md) so we can assign it to a appropriate release

### Code Quality

- [x] Passes `flake8`

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ env:
   global:
     - BUILD_ID=${TRAVIS_BUILD_ID}
     - AIRFLOW_CONTAINER_MOUNT_HOST_VOLUMES="false"
-    - AIRFLOW_CI_SILENT="false"
+    - AIRFLOW_CI_SILENT="true"
```
Comment (kaxil) [inline .travis.yml:25]: Would not that make debugging failures difficult?

Comment (potiuk) [reply .travis.yml:25]: It's mostly for diagnostic messages that were useful at the beginning to see if everything is fine. It's always the question of noise vs. debuggability :(. 

I think I wanted to see the output and decide if it is good to merge this one. I think also when we move to GitLab we will be able to send the verbose output to files and store them as artifacts in GitLab. It's all prepared to do so.

Comment (potiuk) [reply .travis.yml:25]: @kaxil -> take a look at the output. It's fine. We are still using `set +x` in the actual tests so we have very detailed information but in static check SILENT flag will make it much nicer to see the result of static checks directly. It looks good for me now to merge it.

Comment (potiuk) [reply .travis.yml:25]: Yeah. It gets really clumsy without it (we see all files that are checked) so we better merge it.

Comment (kaxil) [reply .travis.yml:25]: 👍

**Pull request merged**